content/docs/command-reference/exp/run.md

```diff
@@ -162,8 +172,7 @@ CPU cores).
   is processed serially).
 
 - `-r <commit>`, `--rev <commit>` - continue an experiment from a specific
-  checkpoint name or hash (`commit`). This is needed for example to resume
-  experiments from `--queue` or `--temp` runs.
+  checkpoint name or hash (`commit`) in `--queue` or `--temp` runs.
```

Comment:
Or maybe

continue a checkpoint experiment run with  `--queue` or `--temp` from a specific checkpoint name or hash (`commit`).

Reply:
Also `--run-all`?

Reply:
The type of experiment/run is "queued experiment" so `--queue` should be clear enough, I think. Sounds like `--run-all` may get extracted to a separate subcommand in the near future anyway.